config/initializers/okcomputer.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
+require 'scholarsphere/redis_config'
+
+redis_config = Scholarsphere::RedisConfig.new
+
 OkComputer.mount_at = false
 
 OkComputer::Registry.register 'solr', OkComputer::SolrCheck.new(Rails.application.config_for(:blacklight)[:url])
+
+OkComputer::Registry.register 'redis', OkComputer::RedisCheck.new(redis_config.call) if redis_config.redis_host
```

Comment:
@whereismyjetpack I think `call` isn't the right name here. Maybe just `config`? Also, you could make a method like `valid?` which returns true or false and checks for the presence of a host, and possibly other things, if needed.

Reply:
i like the idea of valid?